pkg/aws/eks/README.md

```diff
@@ -0,0 +1,23 @@
+# EKS Module
+
+This module is responsible for collecting pricing information for EKS clusters.
+Specifically it collects the pricing information for the following resources:
+- compute instances
+- storage
+
+## Compute Instances
```

Comment:
Let me have a look at this in the morning ... I have some calculations that I wanna try

Reply:
Yeah it's probably the best estimation we can do at the moment ... I will leave this rabbit hole again.